nuke_frontend/src/styles/user-profile.css

```
@@ -587,6 +603,25 @@
+  .up-dossier-row {
+    gap: 8px;
+  }
+  .up-dossier-row__label {
+    flex: 0 0 auto;
+    white-space: nowrap;
+  }
+  .up-dossier-row__value {
+    flex: 1 1 auto;
+    min-width: 0;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)